private/testplanet/satellite.go

```diff
@@ -331,6 +331,9 @@ func (planet *Planet) newSatellites(count int, satelliteDatabases satellitedbtes
 					AuditReputationWeight:       1,
 					AuditReputationDQ:           0.6,
 				},
+				NodeSelectionCache: overlay.CacheConfig{
+					Staleness: time.Hour,
```

Comment:
Use a value that corresponds to more to the production setting (`3 min`) or exercises more code paths `30 sec`.